### DISCUSSION TITLE
covidmeter.org

### DISCUSSION THREAD

**Discussion by zestyping** [microCOVID/microCOVID#1086] [Show and tell]
Hi,  I made this dashboard for my community house in Berkeley.  Perhaps others might find it useful too!  Feel free to share it.

http://covidmeter.org

We have this up on a tablet in the kitchen so we can see it easily when passing by.  I find it convenient for doing quick estimates without having to pull up the microcovid.org calculator.  The numbers are based on those produced by the `update_prevalence` script, though I run it daily rather than weekly.

Ideas?  Suggestions?  Corrections?  All are welcome!

**Comment by blanchardjeremy** [comment]
Oh amazing! That's a nifty little tool. That's helpful for folks to be able to easily put things in context.

PS - our prevalence script now automatically runs daily! 🎉

**Comment by zestyping** [comment]
@blanchardjeremy I just noticed that microcovid.org says the prevalence numbers for Alameda County were last updated on Oct 10 — did one of your jobs stop running?

**Reply by blanchardjeremy** [reply]
@zestyping  -- It was a bug that just got resolved fully this morning. You're good to go now!
